pkgs/servers/sozu/default.nix

```diff
@@ -1,22 +1,26 @@
-{ stdenv, rustPlatform, fetchFromGitHub }:
+{ stdenv, rustPlatform, fetchFromGitHub, darwin }:
 
 rustPlatform.buildRustPackage rec {
-    pname = "sozu";
-    version = "0.11.46";
+  pname = "sozu";
+  version = "0.11.50";
 
-    src = fetchFromGitHub {
-        owner = "sozu-proxy";
-        repo = pname;
-        rev = version;
-        sha256 = "0anng5qvdx9plxs9qqr5wmjjz0gx5113jq28xwbxaaklvd4ni7cm";
-    };
+  src = fetchFromGitHub {
+    owner = "sozu-proxy";
+    repo = pname;
+    rev = version;
+    sha256 = "1srg2b8vwc4vp07kg4fizqj1rbm9hvf6hj1mjdh6yvb9cpbw3jz7";
+  };
 
-    cargoSha256 = "19c2s9h4jk9pff72wdqw384mvrf40d8x4sx7qysnpb4hayq2ijh3";
+  cargoSha256 = "5WOigCiQZQ5DaTd15vV8pUh8Xl3UIe9yLG1ptUtY+iA=";
 
-    meta = with stdenv.lib; {
-        description = "Open Source HTTP Reverse Proxy built in Rust for Immutable Infrastructures";
-        homepage = "https://sozu.io";
-        license = licenses.agpl3;
-        maintainers = with maintainers; [ filalex77 ];
-    };
+  buildInputs = stdenv.lib.optionals stdenv.isDarwin
```

Comment:
```suggestion
  buildInputs = stdenv.lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

Reply:
Thanks, I can't believe I didn't see that!

Reply:
That's not a typo, that's just another method, so you dont need brackets around darwin.apple_sdk.frameworks.Security.

Reply:
Thanks, I didn't know.